IOMC/RandomEngine/python/IOMC_cff.py

```diff
@@ -178,5 +178,14 @@
         initialSeed = cms.untracked.uint32(1234567),
         engineName = cms.untracked.string('HepJamesRandom')) )
 
-eras.phase2_timing.toModify(RandomNumberGeneratorService,
-                            trackTimeValueMapProducer = cms.PSet(initialSeed = cms.untracked.uint32(1234567), engineName = cms.untracked.string('HepJamesRandom') ) )
+eras.phase2_timing.toModify(
```

Comment:
If you save the states of the random engine in the event, it is about 800 bytes per engine per event when using the HepJamesRandom type of engine. 2400 bytes if you instead use TRandom.